contrib/local-environment/Makefile

```diff
@@ -21,3 +21,12 @@ keycloak-up:
 .PHONY: keycloak-%
 keycloak-%:
 	docker-compose -f docker-compose-keycloak.yaml $*
+
+.PHONY: kubernetes-up
+kubernetes-up:
+	cd kubernetes && make create-cluster
+	cd kubernetes && make deploy
```

Comment:
Does `make -C kubernetes` work instead? That's what we have in other placed where we are calling makefiles in different directories